pip/flatpak-pip-generator

```diff
@@ -44,6 +44,10 @@ parser.add_argument('--output', '-o',
                     help='Specify output file name')
 parser.add_argument('--runtime',
                     help='Specify a flatpak to run pip inside of a sandbox, ensures python version compatibility')
+parser.add_argument('--arch-dependent-allow', action='store_true',
+                    help='Allow arch dependent packages to be used')
+parser.add_argument('--arch-dependent-force', action='store_true',
+                    help='Force arch dependent packages to be used')
```

Comment:
README.md should have entries on these new parameters

Reply:
Question: shouldn't this have options to choose which arches are we targeting? Likes of `arches-only` and `skip-arches`?

Otherwise LGTM.

Reply:
> Question: shouldn't this have options to choose which arches are we targeting? Likes of `arches-only` and `skip-arches`?
> 
> Otherwise LGTM.

That seems complicated and a future work. Currently, pip download cannot download a package for another arch unless you specify the whole pep 600 platform tag, eg. `pip download --platform=manylinux_2_31_aarch64 --only-binary=:all: shiboken6`

The tag varies from package to package due to the glibc version/package availability.

There's also an issue where pip download doesn't handle legacy tag aliases too well https://github.com/pypa/pip/issues/10760

I think for now it should be left to host/pip.